### ISSUE TITLE
Thanks for your work

### ISSUE THREAD

**Issue by mr21** [lukehorvat/web-audio-oscillators#1]
Hi :)

Thanks for your work :) I am adding your periodicWaves compilation to @gridsound, some waves are coming from @mohayonao but how did you create the other ones like organ? by brutforce? with an editor?

Also, the `bass.js` and `organ2.js` do not start with a `0` as asked by the specification, and i don't know if I have to add another `0` or if I have to replace the first number by a `0`...

Thanks again anyway

**Comment by lukehorvat** [comment]
> how did you create the other ones like organ?

I brute-forced some, and for others I used the approach detailed in [this article](https://www.sitepoint.com/using-fourier-transforms-web-audio-api/#synthesizing-a-horn).

> Also, the bass.js and organ2.js do not start with a 0 as asked by the specification, and i don't know if I have to add another 0 or if I have to replace the first number by a 0...

Thanks for noticing this. If I'm reading the spec correctly, the first element is always ignored, so I believe "replace" is the path to take here. I've updated them to start with zero in v1.0.4, and there seems to be no difference in sound.

Nice work on GridSound. Looks cool.
